kpet/schema.py

```diff
@@ -395,14 +395,26 @@ def resolve(self, data):
 class List(Node):
     """
     List schema, with every element matching a single specified schema.
+    It supports an optional "min_len" parameter, to force the list to
+    contain at least "min_len" elements.
```

Comment:
I was rather thinking about a docstring for the constructor below (`__init__()`), not adding to the docstring of the class. You can find `__init__()` documented elsewhere in the file for use as a template. Could you do that?